spec/kitchen/transport/base_spec.rb

```diff
@@ -120,4 +120,16 @@
       connection.execute_with_retry("Hi", [123], 3, 1).must_equal "Hello"
     end
   end
+
+  describe "#retry?" do
+    it "raises Invalid Type Error for retry_on_exit_code" do
```

Comment:
is 'Invalid Type Error' left over from a previous iteration?

This could be more more clear as something like `it "raises an exception when multiple retryable exit codes are passed as a string"`.